### DISCUSSION TITLE
How to Move to Unstaged Changes Panel

### DISCUSSION THREAD

**Discussion by duncanam** [jesseduffield/lazygit#3103] [Q&A] [answered]
How do we focus the context to the Unstaged Changes panel on the right? Or in general, how to focus the larger right panel? I can only focus it when I click on it with my mouse. 

I'm probably dumb but a callout in the keybindings for this could be useful, if it's possible.

**Comment by mark2185** [comment, marked as answer]
You can focus it by pressing `Enter`, from the `help` menu (press `?`):

```
Stage individual hunks/lines for file, or collapse/expand for directory
```

If you're talking about switching between the staged and unstaged changes, there is a keybinding in the `help` menu as well:

```
<tab> Switch to other panel (staged/unstaged changes)
```

Is that what you're looking for?

**Reply by duncanam** [reply]
Okay, that's helpful, I read this but interpreted it as it doing something else, like what "Space" does on Panel 2. Again, this was totally me, thank you for clarifying.

**Reply by mark2185** [reply]
Glad to have helped.

Take a look at the available [features](https://github.com/jesseduffield/lazygit#features) of `lazygit`, as well as [15 features in under 15 minutes](https://www.youtube.com/watch?v=CPLdltN7wgE), that could clear more things up.
